.gitignore

```diff
@@ -1,6 +1,8 @@
 ## Ignore Visual Studio temporary files, build results, and
 ## files generated by popular Visual Studio add-ons.
 
+.gitmodules

```

Comment:
there is no point in doing this. .gitignore does *nothing* if the file is already committed.